internal/restic/node.go

```diff
@@ -186,7 +186,13 @@ func (node Node) restoreMetadata(path string) error {
 	var firsterr error
 
 	if err := lchown(path, int(node.UID), int(node.GID)); err != nil {
-		firsterr = errors.Wrap(err, "Lchown")
+		// Like "cp -a" and "rsync -a" do, we only report lchown errors if we run
+		// as root.
+		// On Windows, Geteuid always returns -1, and we always report lchown
+		// errors.
+		if os.Geteuid() <= 0 {
+			firsterr = errors.Wrap(err, "Lchown")
```

Comment:
Hm, how about we also test if `err` is `os.ErrPermission`? Or is this not necessary?

Reply:
Good point. Will do.